### ISSUE TITLE
Index id issues when creating alerts from visualizations (SPY)

### ISSUE THREAD

**Issue by havlan** [sirensolutions/sentinl#452]
1. Sentinl and Kibana version 
Kibana: 6.2.4
Sentinl: 6.2.4 (tag 6.2.3.3)

2. Issue description
When passing alarms directly from visualizations the id of the index is passed and not it's name, causing an error (index_not_found). 

3. Reproducing the issue step-by-step
Using the mos script to generate some test data. (link?)
Visualize count or something. (not vital for the issue i think)
In the spy tab "Set-Alarm", fill in the gaps.
Testing the alarm and receive `Watchers: fail to execute watcher: fail to search: [index_not_found_exception] no such index, with { resource.type="index_or_alias" & resource.id="b61ab270-6e1d-11e8-b83a-ff81cc161f67" & index_uuid="na" & index="b61ab270-6e1d-11e8-b83a-ff81cc161f67" } `

4. Expected results vs. actual results
Actual: [actual.txt](https://gist.githubusercontent.com/havlan/ac06509acdae14c0ac1bd06a8751722a/raw/1304c2d0942df3052530e9cdd10be298b123439e/actual.txt)
Expected: [expected.txt](https://gist.githubusercontent.com/havlan/ac06509acdae14c0ac1bd06a8751722a/raw/1304c2d0942df3052530e9cdd10be298b123439e/expected.txt)

TLDR;
Actual: `"index": [
          "b61ab270-6e1d-11e8-b83a-ff81cc161f67"
        ]`
Expected: `"index": [
          "mos-2018.06.12"
        ]`

5. Sample of data

Script used for generating data to elasticsearch [mos.sh](https://gist.githubusercontent.com/havlan/ac06509acdae14c0ac1bd06a8751722a/raw/1304c2d0942df3052530e9cdd10be298b123439e/mos.sh)

6. Additional details
I am also curious about the possibilty to pass on dynamic time fields to alarms from visualizations, if it's possible at all.

**Comment by havlan** [comment]
![screenshot from 2018-06-12 13-02-09](https://user-images.githubusercontent.com/18334546/41288396-76f18014-6e46-11e8-9135-c67e21cea31d.png)
Should have included this, I added some comments in public/dashboard/alarm_button.js to preview the data. If it's easier to see here, the `SearchRequest.fetchParams.index` is passed as the index to the alarm creation, but that results in an error. `SearchRequest.fetchParams.title` looks like the correct data.

**Comment by lmangani** [comment]
@havlan any chance to test this [PR](https://github.com/sirensolutions/sentinl/pull/453)?

**Comment by havlan** [comment]
@lmangani Will do! :+1:
